src/txmempool.cpp

```diff
@@ -659,9 +659,11 @@ void CTxMemPool::removeForBlock(const std::vector<CTransaction> &vtx,
 {
     LOCK(cs);
     std::vector<CTxMemPoolEntry> entries;
+    int nBlockSize = 0;
     BOOST_FOREACH (const CTransaction &tx, vtx)
     {
         uint256 hash = tx.GetHash();
+        nBlockSize += tx.GetTotalSize();
 
```

Comment:
wouldn't it be better/faster to just get the serialized size of the block rather than calculating each individual transaction and adding them up...also it's a little misleading because it is not really the total blocksize since it does not include the size of the header.

Reply:
the 80 bytes for header is negligible for the calculation the bytes are being used for. I didn't get the block serialize size itself there because the block isn't in scope and we would have to go dig it up. Although you make a good point that as the number of txs per block goes up my code will eventually start to take longer. so yeah. will adjust for that.